src/blocks/PromoFeaturesBlock/PromoFeaturesBlock.scss

```diff
@@ -28,7 +28,7 @@ $block: '.#{$ns}PromoFeaturesBlock';
         background-color: inherit;
 
         &_grey {
-            background-color: var(--yc-color-base-misc);
+            background-color: var(--g-color-base-misc-light);
```

Comment:
We didn't have `--g-color-base-misc` variable? I wonder how this variable may change for dark theme

Reply:
"light" here means "pale" or "not heavy", dark theme supports this color, you can switch to to see it.